vms/platformvm/block/executor/acceptor.go

```diff
@@ -117,57 +117,71 @@ func (a *acceptor) ApricotAtomicBlock(b *block.ApricotAtomicBlock) error {
 }
 
 func (a *acceptor) abortBlock(b block.Block, blockType string) error {
-	parentID := b.Parent()
-	parentState, ok := a.blkIDToState[parentID]
+	blkID := b.ID()
```

Comment:
when we verified the option we copied over option's blockState the preference. Hence we can lookup the option blockState now, instead of its parent's one